Adyen/UI/Form/FormViewController.swift

```diff
@@ -11,42 +11,76 @@ import UIKit
 @_spi(AdyenInternal)
 open class FormViewController: UIViewController, AdyenObserver, PreferredContentSizeConsumer {
 
-    fileprivate enum Animations {
-        fileprivate static let keyboardBottomInset = "keyboardBottomInset"
-        fileprivate static let firstResponder = "firstResponder"
+    private enum Animations {
+        static let keyboardBottomInset = "keyboardBottomInset"
+        static let firstResponder = "firstResponder"
     }
 
+    // MARK: - UI elements
+
+    private let scrollView: UIScrollView = {
+        let scrollView = UIScrollView()
+        scrollView.translatesAutoresizingMaskIntoConstraints = false
+        scrollView.showsVerticalScrollIndicator = true
+        return scrollView
+    }()
+
+    private let formView: FormView = {
+        let form = FormView()
+        form.translatesAutoresizingMaskIntoConstraints = false
+        return form
+    }()
+
+    // MARK: - Public properties
+
     public var requiresKeyboardInput: Bool { formRequiresInputView() }
 
     /// Indicates the `FormViewController` UI styling.
     public let style: ViewStyle
 
     /// Delegate to handle different viewController events.
     public weak var delegate: ViewControllerDelegate?
-    
-    internal lazy var keyboardObserver = KeyboardObserver()
 
-    // MARK: - Public
+    // MARK: - Private properties
+
+    private var keyboardObserver = KeyboardObserver()
+    private var scrollEnabled: Bool
+
+    // MARK: - Initializers
 
     /// Initializes the FormViewController.
     ///
     /// - Parameters:
+    ///   - scrollEnabled: Boolean value that determines whether the form view contains a scroll view in its view hierarchy.
     ///   - style: The `FormViewController` UI style.
     ///   - localizationParameters: The localization parameters.
     public init(
+        scrollEnabled: Bool,
```

Comment:
Using composition would make this property redundant. We will extract this logic to an assembly layer.